corpus/accounts/models.py

```diff
@@ -40,15 +41,111 @@ def users(self):
 
 
 class User(AbstractUser):
+    GENDERS = [
+        ("M", "Male"),
+        ("F", "Female"),
+        ("O", "Other"),
+        ("N", "Prefer not to disclose"),
+    ]
+
     username = None
-    phone_no = models.CharField(max_length=13, unique=True, blank=False, null=False)
-    gender = models.CharField(max_length=1, choices=GENDERS, blank=False, null=False)
-    email = models.EmailField(unique=True, blank=False, null=False)
+    phone_no = models.CharField(
+        max_length=15,
+        unique=True,
+        verbose_name="Phone Number",
+        validators=[validate_phone_number],
+    )
+    gender = models.CharField(max_length=1, choices=GENDERS)
+    email = models.EmailField(unique=True, verbose_name="Personal Email")
```

Comment:
For non executive members, we shouldn't care about what email address people fill. People can fill in whatever email address they want, its only for executive members (and events if required) where we need to enforce specific email addresses.

If I wasn't involved with IEEE and just wanted to create an account, I'd probably make it with my EDU account so that I'm sure my personal email (and thus personal data) remains safe.